Documentation/building/unix-instructions.md

```diff
@@ -12,6 +12,9 @@ Only use it when the parameters that you are passing to the script apply for bot
 
 For more information about the different options when building, run `build.sh -?` and look at examples in the [developer-guide](../project-docs/developer-guide.md).
 
+## Environment
+An environment with at least 2GB RAM is required to complete the managed build.
```

Comment:
We already have different sections for the native and managed builds. Maybe we can just put this under the managed section?